### DISCUSSION TITLE
More configuration options for markdown column

### DISCUSSION THREAD

**Discussion by ziming** [Laravel-Backpack/community-forum#1245] [Ideas]
@tabacitu  asked me to create it here when I emailed about it earlier this month, @pxpm is cced in that email later as well so you can refer to that email

So basically more options can be passed to laravel Str::markdown() method to strengthen or weaken the security of markdown parsing, it will be good if the markdown column allow us to pass in the extra options too

Thank you

**Comment by pxpm** [comment]
I've already read it @ziming sorry I haven't got time to get to it, (and to be honest, I totally forgot about it). 
Thanks or opening the issue here, I will comment here so that it appears on my "participating" notifications. I will do it and ping you here when I do, but probably expect it to be launch alongside v7 that we are already working on 👍

**Comment by pxpm** [comment]
This totally slipped through the cracks. But I found it, and did it. 

Markdown column now support additional options, and was made "secure" by default. 

Docs have already been updated and I will release a version with the fixes later today 👍
